.github/workflows/publish.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish and Release
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+
+jobs:
+  publish:
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.head.ref, 'release/')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
```

Comment:
Can we add a check that makes this step fail if `npm audit` fails? It would be nice to ensure that we always publish with up to date dependencies